scripts/storage/qcow2/managesnapshot.sh

```diff
@@ -153,7 +153,7 @@ destroy_snapshot() {
     lvm lvremove -f "${vg}/${snapshotname}-cow"
   elif [ -f $disk ]; then
      #delete all the existing snapshots
-     $qemu_img snapshot -l $disk |tail -n +3|awk '{print $1}'|xargs -I {} $qemu_img snapshot -d {} $disk >&2
+     $qemu_img snapshot -l $disk |tail -n +3|awk '{print $2}'|xargs -I {} $qemu_img snapshot -d {} $disk >&2
```

Comment:
@shwstppr @davidjumani can any of you advise if `qemu-img snapshot -l` second column is the name of the snapshot or ID? qemu-img snapshot -d in Ubuntu 20.04 and possibly CentOS8 only accepts snapshot name and not the ID.

Reply:
@rhtyd from [the qemu wiki](https://wiki.qemu.org/Features/SnapshottingImprovements) :
```
$ qemu-img snapshot -l xxtest.qcow2
Snapshot list:
ID        TAG                 VM SIZE                DATE       VM CLOCK
1                                1.5M 2010-07-26 16:51:52   00:00:08.599
2                                1.5M 2010-07-26 16:51:53   00:00:09.719
3                                1.5M 2010-07-26 17:26:49   00:00:13.245
4                                1.5M 2010-07-26 19:01:00   00:00:46.763
```
no name to be found.

Reply:
@DaanHoogland I think we'll have to check the command for our snapshots, when we create them we generally pass like a uuid-looking name. @shwstppr @davidjumani can you test and confirm (with centos7 vs centos8) with this PR?